frontend/cypress/integration/integration_spec.js

```diff
@@ -0,0 +1,74 @@
+import authDetails from "../support/authDetails.json";
+describe("Stam Acasa", function() {
+  const BEARER_TOKEN = "something-really-secret";
+  const mockLogin = () => {
+    authDetails.access_token = BEARER_TOKEN;
+    window.sessionStorage.setItem(
+      "oidc.user:http://localhost:5001:js",
+      JSON.stringify(authDetails)
+    );
+  };
+
+  describe("Main logged in journeys", () => {
+    const stubProfile = data => {
+      cy.routeUseAuth(
+        {
+          method: "GET",
+          url: "http://localhost:5008/api/profile",
```

Comment:
Could we use the env variables [like here](https://github.com/code4romania/stam-acasa/blob/develop/frontend/src/api/api.js#L6) otherwise the stubs will be bypassed if we change the URLs in the min app ?